Backtesting/collectors/exchanges/exchange_bot_snapshot_data_collector/bot_snapshot_with_history_collector.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, config, exchange_name, tentacles_setup_config, symbols, time_
         self.set_file_path()
 
     def get_permanent_file_identifier(self):
-        symbols = "-".join(symbol_util.merge_symbol(symbol) for symbol in self.symbols)
+        symbols = "-".join(symbol_util.merge_symbol(symbol.symbol_str).replace(":", "_") for symbol in self.symbols)
```

Comment:
I think we should put this conversion in symbol_util

Reply:
I'm not sure as this is a specific conversion to create the backtesting file name only, it's only used here